### PR TITLE
Adds omniauth (google so far) features for registered users

### PR DESCRIPTION
**Pull request by FloHin**
Adds the following:
- Registered users may connect/link omniauth providers in their control panel
- Afterwards they may sign in directly per omniauth

Details:
- uses Identity-pattern to leave User model as it is
- just google so far

Added: 
- text and translations

### CODE REVIEW

**Comment by frenkel** [comment]
Thank you very much! Could you fix the comments of the hound? You can ignore the double quote advise. I like your style more.

File: app/controllers/omniauth_controller.rb [hunk 1]
```diff
@@ -0,0 +1,59 @@
+class OmniauthController < ApplicationController
+
+  skip_before_filter :verify_authenticity_token, only: :callback
+  skip_before_filter :authenticate_user, only: :callback
+  skip_authorization_check :only => :callback
+
+  def callback
```
Comment (FloHin) [inline app/controllers/omniauth_controller.rb]: What does houndci need in order to improve this code?

**Comment by FloHin** [comment]
The new commits add translations for german and english and should fix the hound comments (except double-quotes)

**Comment by frenkel** [comment]
I've reviewed much of the pull request. It seems you implemented it in a non-standard manner, therefore you needed to manually add routes (instead of them being auto-generated). Also, you didn't inherit from the correct OmniAuth DeviseController, but instead chose ApplicationController, which resulted in all kinds of access problems with cancan and devise (all changes in ApplicationController could have been avoided). I've reworked those problems and merged all the code. Thanks a lot for this contribution, I think a lot of users will really like this!

**Pull request closed without merging**